lib/servers/standalone.js

```diff
@@ -26,12 +26,21 @@ class StandaloneServer {
 
     // POST /session
     if(this._isCreate(req.method, req.url)) {
-      var sess = this._manager.createSession();
-      this.logger.info('session '+sess.id+' created');
+      var content = "";
+      req.setEncoding("utf8");
```

Comment:
is this a valid encoding for all use cases?

Reply:
Most but not all. Per [RFC 7159](https://tools.ietf.org/html/rfc7159#section-8.1): 
```
   JSON text SHALL be encoded in UTF-8, UTF-16, or UTF-32.  The default
   encoding is UTF-8, and JSON texts that are encoded in UTF-8 are
   interoperable in the sense that they will be read successfully by the
   maximum number of implementations; there are many implementations
   that cannot successfully read texts in other encodings (such as
   UTF-16 and UTF-32).

   Implementations MUST NOT add a byte order mark to the beginning of a
   JSON text.  In the interests of interoperability, implementations
   that parse JSON texts MAY ignore the presence of a byte order mark
   rather than treating it as an error.
```

Reply:
I see. For some reason I was forgetting that this is just for the "POST" response